tools/buildsys/src/builder.rs

```diff
@@ -511,7 +626,20 @@ fn secrets_args() -> Result<Vec<String>> {
         );
     }
 
-    for var in &[
+    let ca_bundle_var = "BUILDSYS_CACERTS_BUNDLE_OVERRIDE";
+    let ca_bundle_value =
+        env::var(ca_bundle_var).context(error::EnvironmentSnafu { var: ca_bundle_var })?;
+
+    if !ca_bundle_value.is_empty() {
+        let ca_bundle_path = PathBuf::from(&ca_bundle_value);
+        if ca_bundle_path.exists() {
+            args.build_secret("file", "ca-bundle.crt", &ca_bundle_path.to_string_lossy());
+        } else {
+            return error::BadCaBundleSnafu { ca_bundle_path }.fail();
+        }
+    }
```

Comment:
nit: better style is to invert the check and early return to reduce excessive indentation

```suggestion
        if !ca_bundle_path.exists() {
            return error::BadCaBundleSnafu { ca_bundle_path }.fail();
        }
        
        args.build_secret("file", "ca-bundle.crt", &ca_bundle_path.to_string_lossy());
    }
```